### ISSUE TITLE
Reproduce Issues

### ISSUE THREAD

**Issue by veophi** [benedekrozemberczki/CapsGNN#11]
Hi, thanks for your PyTorch codes of GapsGNN. I try to run the codes on NCI, DD, and other graph classification datasets, but it doesn't  work (For example, training loss converges to 2.0, and test acc is about 50% on NCI1 after several iterations.) 
How should I do if I want to run these codes on NCI, DD and etc?
Thanks again.

**Comment by benedekrozemberczki** [comment]
Versions are updated.
